### DISCUSSION TITLE
How to connect to Soketi server from a Websocket GUI client like Insomnia, postman ...

### DISCUSSION THREAD

**Discussion by fikoulli** [soketi/pws#856] [Q&A]
Hello,

Plz,
How to connect to Soketi server from a Websocket GUI client like Insomnia, postman ...

Thanks

**Comment by petczu** [comment]
Hi @fikoulli 

In the last version of Postman, you can create a new Request and choose WebSocket type
Next you need to provide URL wss://server:port/app/your-app-key
Change only host, port and your-app-key
